[2.0.X] MKS Sbase compile error with SDSUPPORT

I am using the latest atom, platformio and the latest 2.0.0 source code.
When I compile with `SDSUPPORT` I get an error:
```
Marlin\src\sd\cardreader.cpp: In member function 'void CardReader::initsd()':
Marlin\src\sd\cardreader.cpp:245:29: error: 'SDSS' was not declared in this scope
if (!card.init(SPI_SPEED, SDSS)
^~~~
*** [.pioenvs\LPC1768\src\src\sd\cardreader.o] Error 1
 [ERROR] Took 211.57 seconds
```
Could somebody help me how/where to declare this SDSS?

--- a/Marlin/src/HAL/HAL_LPC1768/spi_pins.h
+++ b/Marlin/src/HAL/HAL_LPC1768/spi_pins.h
@@ -27,45 +27,48 @@
 
 #if MB(MKS_SBASE)
 
-#define LPC_SOFTWARE_SPI  // MKS_SBASE needs a software SPI because the
-                          // selected pins are not on a hardware SPI controller
+  #define LPC_SOFTWARE_SPI  // MKS_SBASE needs a software SPI because the
+                            // selected pins are not on a hardware SPI controller
 
-// A custom cable is needed. See the README file in the
-// Marlin\src\config\examples\Mks\Sbase directory
+  // A custom cable is needed. See the README file in the
+  // Marlin\src\config\examples\Mks\Sbase directory
 
-#define SCK_PIN           P1_22  // J8-2 (moved from EXP2 P0.7)
-#define MISO_PIN          P1_23  // J8-3 (moved from EXP2 P0.8)
-#define MOSI_PIN          P2_12  // J8-4 (moved from EXP2 P0.5)
-#define SS_PIN            P0_28
+  #define SCK_PIN           P1_22  // J8-2 (moved from EXP2 P0.7)
+  #define MISO_PIN          P1_23  // J8-3 (moved from EXP2 P0.8)
+  #define MOSI_PIN          P2_12  // J8-4 (moved from EXP2 P0.5)
+  #define SS_PIN            P0_28
+  #define SD_DETECT_PIN     P0_27
 
 #else
 
-#define LPC_SOFTWARE_SPI  // Re-ARM board needs a software SPI because using the
-                          // standard LCD adapter results in the LCD and the
-                          // SD card sharing a single SPI when the RepRap Full
-                          // Graphic Smart Controller is selected
+  #define LPC_SOFTWARE_SPI  // Re-ARM board needs a software SPI because using the
+                            // standard LCD adapter results in the LCD and the
+                            // SD card sharing a single SPI when the RepRap Full
+                            // Graphic Smart Controller is selected
 
-/** onboard SD card */
-//#define SCK_PIN           P0_07
-//#define MISO_PIN          P0_08
-//#define MOSI_PIN          P0_09
-//#define SS_PIN            P0_06
-/** external */
-#ifndef SCK_PIN
-  #define SCK_PIN           P0_15
-#endif
-#ifndef MISO_PIN
-  #define MISO_PIN          P0_17
-#endif
-#ifndef MOSI_PIN
-  #define MOSI_PIN          P0_18
-#endif
-#ifndef SS_PIN
-  #define SS_PIN            P1_23
-#endif
+  /** onboard SD card */
+  //#define SCK_PIN           P0_07
+  //#define MISO_PIN          P0_08
+  //#define MOSI_PIN          P0_09
+  //#define SS_PIN            P0_06
+  /** external */
+  #ifndef SCK_PIN
+    #define SCK_PIN           P0_15
+  #endif
+  #ifndef MISO_PIN
+    #define MISO_PIN          P0_17
+  #endif
+  #ifndef MOSI_PIN
+    #define MOSI_PIN          P0_18
+  #endif
+  #ifndef SS_PIN
+    #define SS_PIN            P1_23
+  #endif
+
+#endif // MKS_SBASE
+
 #ifndef SDSS
   #define SDSS              SS_PIN
 #endif
 
-#endif // MKS_SBASE
-#endif /* SPI_PINS_LPC1768_H */
+#endif // SPI_PINS_LPC1768_H

